keps/sig-node/2371-cri-pod-container-stats/README.md

```diff
@@ -304,7 +304,7 @@ These correspond to some fields of the [ContainerStats](#summary-container-stats
 +    // Corresponds to Stats Summary API CPUStats UsageCoreNanoSeconds
 =    UInt64Value usage_core_nano_seconds = 2;
 +    // Total CPU usage (sum of all cores) averaged over the sample window.
-+    UInt64Value usage_nano_seconds = 3;
++    UInt64Value usage_nano_cores = 3;
```

Comment:
this is the actual variable name

Reply:
and matches summary API naming:

https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/kubelet/pkg/apis/stats/v1alpha1/types.go#L205-L208

Reply:
```The "core" unit can be interpreted as CPU core-nanoseconds per second```
maybe  add ^ that sentence...